### DISCUSSION TITLE
[Dropdown] display with button item show transition on left or right element

### DISCUSSION THREAD

**Discussion by apple-idn** [fomantic/Fomantic-UI#793] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Dropdown with button right or left on show

## Problem
how dropdown with button item and left or right show 
##Expected
![expected](https://user-images.githubusercontent.com/33564576/58849409-d2325b80-86bc-11e9-9adb-5ac409039da7.jpg)
##result
<img width="161" alt="result" src="https://user-images.githubusercontent.com/33564576/58849423-d9596980-86bc-11e9-9e05-564c7bd0f6ec.png">

https://jsfiddle.net/allwe/6h1geLuv/7/

**Comment by lubber-de** [comment, marked as answer]
You can use `left pointing dropdown` and remove the pointer by
```css
.custom5 .menu:after {
  opacity: 0;
}
```

See https://jsfiddle.net/k1hr9c7o/

**Comment by lubber-de** [comment]
@apple-idn Does my suggestion work for you? Can we close this?

**Comment by lubber-de** [comment]
Closing because of no more response and a provided solution,
